src/main/Config.cpp

```diff
@@ -1074,9 +1074,9 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
             {
                 EXPERIMENTAL_BACKGROUND_OVERLAY_PROCESSING = readBool(item);
             }
-            else if (item.first == "EXPERIMENTAL_BACKGROUND_EVICTION_SCAN")
+            else if (item.first == "BACKGROUND_EVICTION_SCAN")
             {
-                EXPERIMENTAL_BACKGROUND_EVICTION_SCAN = readBool(item);
+                BACKGROUND_EVICTION_SCAN = readBool(item);
```

Comment:
should we add a warning here if BucketListDB isn't enabled?

Reply:
We do, warning is on `src/main/ApplicationImpl.cpp:836`

Reply:
Oh i see, should we throw though? It's likely operators won't notice the misconfiguration otherwise

Reply:
I'd rather not. Bot this flag and the SQL flag are going away next release anyway, and I doubt anyone would have a reason to set this flag manually. Even if they did, currently there's essentially no difference between foreground and background eviction (because the scan size is still small), so I'd rather not crash over something so minor.

Reply:
well it's a crash on startup right after an operator manually makes a change to the config, so crashing seems more reasonable than silently proceeding, as it indicates that there's a misconfiguration (that's typically what we do for other configs that can't co-exist together). otherwise, operators have no idea that their nodes are misconfigured and get bitten by it later.

Reply:
Done

Reply:
thanks!